Source/WebCore/platform/ScrollableArea.h

```diff
@@ -394,6 +395,7 @@ class ScrollableArea : public CanMakeWeakPtr<ScrollableArea> {
     bool shouldBlockScrollPropagation(const FloatSize&) const;
     FloatSize deltaForPropagation(const FloatSize&) const;
     WEBCORE_EXPORT virtual float adjustVerticalPageScrollStepForFixedContent(float step);
+    virtual bool needsAnimatedScroll() const { return false; }
```

Comment:
This isn't "needs", it's more like "isRunningAnimatedScroll" I think. Why isn't this deducible from `ScrollAnimationStatus` already?

Reply:
This bool is only set for this case so its not deducible from ScrollAnimationStatus.